suite-native/module-onboarding/src/screens/AnalyticsConsentScreen.tsx

```diff
@@ -96,6 +96,7 @@ export const AnalyticsConsentScreen = ({
                                         }
                                     />
                                 </VStack>
+                                {/* Todo: this shall probably use `TouchableSwitchRow` */}
```

Comment:
Could be changed straight away if it's same component? If you want, just ping me and I'll commit the change, but I hate just adding todo's for later. They tend to rot in the code for years :)